src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -1,68 +1,39 @@
 package com.kodilla.ecommercee.domain;
 
 import lombok.AllArgsConstructor;
+import lombok.Data;
 import lombok.NoArgsConstructor;
 
 import javax.persistence.*;
 import javax.validation.constraints.NotNull;
 import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
 
 @Entity(name = "carts")
-@Table
-@AllArgsConstructor
 @NoArgsConstructor
+@AllArgsConstructor
+@Data
 public class Cart {
-    private int id;
-    private String name;
-    private String description;
-    private BigDecimal price;
-    private int groupId;
-
     @Id
-    @GeneratedValue
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     @NotNull
-    @Column(name = "Id", unique = true)
-    public int getId() {
-        return id;
-    }
-
-    public void setId(int id) {
-        this.id = id;
-    }
+    @Column(name = "id", unique = true)
+    private Long id;
 
     @Column(name = "name")
-    public String getName() {
-        return name;
-    }
-
-    public void setName(String name) {
-        this.name = name;
-    }
+    private String name;
 
     @Column(name = "description")
-    public String getDescription() {
-        return description;
-    }
-
-    public void setDescription(String description) {
-        this.description = description;
-    }
+    private String description;
 
     @Column(name = "price")
-    public BigDecimal getPrice() {
-        return price;
-    }
-
-    public void setPrice(BigDecimal price) {
-        this.price = price;
-    }
-
-    @Column(name = "group_id")
-    public int getGroupId() {
-        return groupId;
-    }
+    private BigDecimal price;
```

Comment:
I znów jak wyżej. :-D Czy to cena koszyka, czy cena produktu/ów, które w koszyku się znajdują? Możesz zastanowić się na dodaniem np. ilości produktów danego typu w koszyku (burak - 100 szt), sumarycznej kwoty za daną ilość produktów oraz podsumowanie kwoty całego koszyka.